style.css

```diff
@@ -54,7 +54,7 @@ body, html {
     }
   
     50% {
-      transform: translateX(800px);
+      transform: translateX(-100px);
```

Comment:
Wow! :+1:, Nice using the transform. I think using duplicate styling will affect the layout of your document. Coud you consider removing the duplicates?